README.md

```diff
@@ -1,28 +1,12 @@
 ![Arrow Banner](https://github.com/Arrow-air/.github/raw/main/profile/assets/arrow_v2_twitter-banner_neu.png)
 
-# MODULE_NAME Service
-
-*TODO after cloning:*
-
-1. *Replace the repository name for each:*
-
-![Rust
```

Comment:
Keep the badges, just replace the svc-template-rust part

Reply:
Ok, got it now. I think I removed badges on svc-scheduler too